src/components/ClientCard/ClientCard.jsx

```diff
@@ -22,17 +26,20 @@ const ClientCard = props => (
 
 ClientCard.propTypes = {
   chat: PropTypes.object.isRequired,
+  config: PropTypes.object.isRequired,
   children: PropTypes.node.isRequired,
   setActiveChat: PropTypes.func.isRequired,
+  loadMessages: PropTypes.func.isRequired,
 };
 
 
-const mapStateToProps = (/* state */) => ({
-
+const mapStateToProps = state => ({
+  config: state.chat.config,
 });
 
 const mapDispatchToProps = dispatch => ({
   setActiveChat: chat => dispatch(setActiveChat(chat)),
+  loadMessages: (config, id) => loadMessages(dispatch, config, id),
```

Comment:
the reason we have to pass dispatch into the function is because it's an async action creator ... if we install `redux-thunk` then it will look the same as all other actions creators. Bronek and I decided to write it vanilla until we felt the need to include another library / refactor